test/functional/feature_dst20.py

```diff
@@ -488,7 +489,7 @@ def run_test(self):
         )
 
         # Contract ABI
-        self.abi = open("./lib/ain-contracts/dst20/output/abi.json", "r", encoding="utf8").read()
+        self.abi = open(f"{os.path.dirname(__file__)}/../../lib/ain-contracts/dst20/output/abi.json", "r", encoding="utf8").read()
```

Comment:
ok, to avoid these, we do need to use `OUT_DIR` convention and build scripts need to use this as well.

https://doc.rust-lang.org/cargo/reference/environment-variables.html 

We already do this in other build scripts. ain-contracts will have to start using CARGO_MANIFEST_DIR for source, and OUT_DIR for target.

Reply:
TODO later